lib/Backend/Lower.cpp

```diff
@@ -7692,6 +7684,17 @@ Lowerer::LowerUnaryHelperMemWithBoolReference(IR::Instr *instr, IR::JnHelperMeth
     return this->LowerUnaryHelperMem(instr, helperMethod, useBoolForBailout ? boolOpnd : nullptr);
 }
 
+IR::Instr *
+Lowerer::LowerInitCachedScope(IR::Instr* instr)
+{
+    instr->m_opcode = Js::OpCode::CallHelper;
```

Comment:
This will be lowerer during the subsequent iteration of Instructions.

---
In reply to: [67748513](https://github.com/Microsoft/ChakraCore/pull/1135#discussion_r67748513) [](ancestors = 67748513)